core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.cpp

```diff
@@ -113,7 +113,9 @@ void OpenMPTargetExec::resize_scratch(int64_t team_size, int64_t shmem_size_L0,
   // max_active_teams is the number of active teams on the given hardware.
   // We set the number of teams to be twice the number of max_active_teams for
   // the compiler to pick the right number in its case.
+#if !defined(KOKKOS_COMPILER_CRAY_LLVM)
```

Comment:
Can you duplicate the FIXME you added in `Kokkos_OpenMPTarget_Insstance.cpp` (// FIXME_OPENMPTARGTE: Cray compiler did not yet implement omp_set_num_teams) here?